.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 sudo: false
 python:
-    - '2.6'
     - '2.7'
```

Comment:
Sigh in which direction? Django dropped 2.6 support in version 1.7 and refactoring to use Python 3 is a bit too big an undertaking at the moment.